portfolio/src/main/java/com/google/sps/servlets/models/PubSubHandler.java

```diff
@@ -74,16 +58,17 @@ public void onFailure(Throwable throwable) {
           if (throwable instanceof ApiException) {
             ApiException apiException = ((ApiException) throwable);
             // details on the API exception
-            System.out.println(apiException.getStatusCode().getCode());
-            System.out.println(apiException.isRetryable());
+            System.out.println(LOG_CONTEXT + "Failure;");
```

Comment:
In the future, consider making (or reusing) a logger that will automatically append contextual information for you using reflection.